### PR TITLE
INSTALL.md update and .yml for boost-test installation.

### PR DESCRIPTION
**Pull request by jmkerloch**

### CODE REVIEW

File: .github/workflows/linux-system.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       run: |
            sudo apt-get update --fix-missing
            sudo apt-get install libuuid1 uuid-dev libssh2-1 libssh2-1-dev libidn2-dev libidn11 libidn11-dev gtk2.0 libwxgtk3.0-dev libcurl4-openssl-dev libb64-dev
-           sudo apt-get install libboost-all-dev
+           sudo apt-get install libboost-test-dev
```
Comment (sgatto) [inline .github/workflows/linux-system.yml:25]: très bien, ça devrait aller plus vite

File: INSTALL.md [hunk 2]
```diff
@@ -72,10 +109,10 @@ For Windows we will use [vcpkg](https://github.com/microsoft/vcpkg) to download
...
 You must install the corresponding [vcpkg-triplet](https://vcpkg.readthedocs.io/en/latest/users/integration/#triplet-selection) depending on Antares version and libraries load:
 
-- x64-windows        : 64 bits version with dynamic librairies load
-- x86-windows        : 32 bits version with dynamic librairies load
-- x64-windows-static : 64 bits version with static librairies load
-- x86-windows-static : 32 bits version with static librairies load
+- ``x64-windows``        : 64 bits version with dynamic librairies load
+- ``x86-windows``        : 32 bits version with dynamic librairies load
+- ``x64-windows-static`` : 64 bits version with static librairies load
+- ``x86-windows-static`` : 32 bis version with static librairies load
```
Comment (sgatto) [inline INSTALL.md]: `32 bits`
à la place de
`32 bis `

**Review by sgatto — APPROVED** [review]

File: INSTALL.md [hunk 3]
```diff
@@ -136,116 +177,109 @@ cmake -DCMAKE_PREFIX_PATH=<previous_build_dir>/dependencies/install
...
 Antares source directory is named `[antares_src]` in all commands.
 
-Build can be done 'out of source'.
+Note :
+> cpack NSIS installer creation need a 'out of source build'. The build directory must be outside `[antares_src]` directory
+
+### Complete solution including GUI
 
-You can define build type with ```-DCMAKE_BUILD_TYPE``` option.
+You can define build type with ```-DCMAKE_BUILD_TYPE``` and ```--config``` option.
```
Comment (guilpier-code) [inline INSTALL.md:188]: You can define build mode with ```-DCMAKE_BUILD_TYPE``` (configuration step) and ```--config``` (build step) options.

File: INSTALL.md [hunk 4]
```diff
@@ -136,116 +177,109 @@ cmake -DCMAKE_PREFIX_PATH=<previous_build_dir>/dependencies/install
...
- debug
+-  debug
  ```
-cmake -DCMAKE_BUILD_TYPE=debug ..
+cmake -B _build -S [antares_src] -DCMAKE_BUILD_TYPE=debug
+cmake --build _build --config debug
 ```
-Note that these are not the standard CMAKE_BUILD_TYPE. CMake files must be updated.
+Note :
+> These are not the standard ``CMAKE_BUILD_TYPE``. CMake files must be updated.
```
Comment (guilpier-code) [inline INSTALL.md:202]: If option ``CMAKE_BUILD_TYPE`` does not exist for your CMake, update it.

File: INSTALL.md [hunk 5]
```diff
@@ -136,116 +177,109 @@ cmake -DCMAKE_PREFIX_PATH=<previous_build_dir>/dependencies/install
...
 ```
-cd [antares_src]
-cd _build
-cmake -DBUILD_DEPS=ON ..
+cmake -B _build -S [antares_src] -DBUILD_DEPS=ON -DCMAKE_BUILD_TYPE=release ..
 ```
 - Build
  ```
-cd [antares_src]
+cmake --build _build --config release -j8
```
Comment (guilpier-code) [inline INSTALL.md:246]: Not sure --config release is necessary if -DCMAKE_BUILD_TYPE=release was given at configuration time

File: INSTALL.md [hunk 6]
```diff
@@ -72,10 +109,10 @@ For Windows we will use [vcpkg](https://github.com/microsoft/vcpkg) to download
 
 You must install the corresponding [vcpkg-triplet](https://vcpkg.readthedocs.io/en/latest/users/integration/#triplet-selection) depending on Antares version and libraries load:
```
[the remark this answers is not in the document]
Comment (jmkerloch) [reply INSTALL.md:110]: I think it's best to use always last available stable version.

File: INSTALL.md [hunk 7]
```diff
@@ -136,116 +177,109 @@ cmake -DCMAKE_PREFIX_PATH=<previous_build_dir>/dependencies/install
 ````
 
 ## [Building Antares Solution](#build)
-### Complete solution including GUI
+
 Antares source directory is named `[antares_src]` in all commands.
 
-Build can be done 'out of source'.
+Note :
+> cpack NSIS installer creation need a 'out of source build'. The build directory must be outside `[antares_src]` directory
```
[the remark this answers is not in the document]
Comment (jmkerloch) [reply INSTALL.md:184]: Build can be done in almost any directory.
I've just updated the branch with  examples in NSIS to be able to do that.

File: INSTALL.md (hunk 4 above)
Comment (jmkerloch) [reply INSTALL.md:202]: This is not what I meant.
I'm saying that we are not using standard CMAKE_BUILD_TYPE. A feature must be developped to support standard values (Release Debug)

Comment (guilpier-code) [reply INSTALL.md:202]: ok

**Review by guilpier-code — APPROVED** [review]

**Pull request merged**